pkg/models/metering/type.go

```diff
@@ -251,13 +315,28 @@ func (ds *DaemonsetStatistic) Aggregate() {
 }
 
 type ResourceStatistic struct {
-	Apps         map[string]*AppStatistic         `json:"apps" description:"app statistic"`
-	Services     map[string]*ServiceStatistic     `json:"services" description:"service statistic"`
+	// op statistic
+	Ops map[string]*OpStatistic `json:"ops" description:"op statistic"`
```

Comment:
Should use the full name `openpitrix` instead of `op`

Reply:
got it